command/job_history_test.go

```diff
@@ -166,3 +166,209 @@ namespace "default" {
 		})
 	}
 }
+
+func TestJobHistoryCommand_Diffs(t *testing.T) {
+	ci.Parallel(t)
+
+	// Start test server
+	srv, _, url := testServer(t, true, nil)
+	defer srv.Shutdown()
+	state := srv.Agent.Server().State()
+
+	// Create a job with multiple versions
+	job := mock.Job()
+
+	job.ID = "test-job-history"
+	job.TaskGroups[0].Count = 1
+	must.NoError(t, state.UpsertJob(structs.MsgTypeTestSetup, 1000, nil, job))
+
+	v2 := job.Copy()
```

Comment:
This test might be a little clearer if `job` was called `v0` and then this would be `v1`.